great_expectations/expectations/expectation.py

```diff
@@ -174,12 +175,16 @@ class MetaExpectation(ABCMeta):
     attribute, or, if that is not set, by snake-casing the name of the class.
     """
 
+    default_kwarg_values: Dict[str, Optional[Any]] = {}
```

Comment:
```suggestion
    default_kwarg_values: Dict[str, Any] = {}
```

Reply:
I was able to use `object`, thanks for the suggestion!

Reply:
See my comment on line 995 about the downstream effects.